From 3ee0bb375a434558b9a0ef7c91ca23bbf64ff8c3 Mon Sep 17 00:00:00 2001
From: Philippe Gerum <rpm@xenomai.org>
Date: Sat, 22 Jun 2019 19:26:00 +0200
Subject: [PATCH 0453/1010] evl: add explicit requirement for DEVTMPFS

We create devices on the fly as part of the procedure for
instantiating new EVL elements, expecting them to appear dynamically
in the /dev filesystem hierarchy.

Signed-off-by: Philippe Gerum <rpm@xenomai.org>
---
 kernel/Kconfig.evl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/Kconfig.evl b/kernel/Kconfig.evl
index b9ab8aeb54..668e5d56f9 100644
--- a/kernel/Kconfig.evl
+++ b/kernel/Kconfig.evl
@@ -7,6 +7,7 @@ menuconfig EVL
 	bool "EVL real-time core"
 	depends on HAVE_ARCH_EVL
 	select DOVETAIL
+	select DEVTMPFS
 	help
 
 	  The EVL core is a real-time component of the Linux kernel,
-- 
2.30.2

